master/pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.sqlfluff.core]
+templater = "jinja"
+sql_file_exts = ".sql,.sql.j2,.dml,.ddl"
+exclude_rules = "LT01,L034"
```

Comment:
it will result in a lot of other changes some of which didn't look good to me but the consistency it adds would be still valuable. I'll push a commit to show what it looks like

Reply:
https://github.com/determined-ai/determined/pull/6414/commits/ffd29c0e61812e8c10d7e04774867ff3792fc18d